src/main/java/csv/PersonCSV.java

```diff
@@ -0,0 +1,32 @@
+package main.java.csv;
+
+import java.io.Serializable;
+
+public class PersonCSV implements Serializable {
+    private final String name;
+    private final String age;
+    private final String city;
+
+    public PersonCSV(String name, String age, String city) {
+        this.name = name;
+        this.age = age;
+        this.city = city;
+
+    }
+
+    public String PersonToString(PersonCSV person) {
```

Comment:
Check the Java conventions for naming:
- variables
- methods
- classes
- constant variables
- class files